src/lycophron/cli.py

```diff
@@ -92,6 +93,194 @@ def configure():
     pass
 
 
+@lycophron.command()
+@click.option(
+    "--custom",
+    type=str,
+    help="Custom field namespaces separated by commas (e.g., dwc,ac,obo)"
+)
+@click.option(
+    "--filename",
+    type=click.File(mode="w", lazy=True),
+    default="data.csv",
+    help="Output CSV filename",
+)
+@click.option("--all", is_flag=True, help="Include all fields")
+def new(
+    custom,
+    filename,
+    all,
+):
+    """Creates a new CSV template."""
+    # Base RDM fields
+    rdm_fields = [
```

Comment:
major: I think it's missing the `filename` field, to specify the name of the file to be uploaded.